docs/data/material/migration/upgrade-to-v7/upgrade-to-v7.md

```diff
@@ -273,7 +266,8 @@ function ColorModeToggle() {
   }, [mode]);
 
   React.useEffect(() => {
-    console.log(theme.palette.mode); // logged 'light' at first render, no log after the button click
+    // logged 'light' at first render, no log after the button click
+    console.log(theme.palette.mode);
```

Comment:
Fix the horizontal scrollbar https://mui.com/material-ui/migration/upgrade-to-v7/#theme-behavior-changes 

<img width="855" alt="SCR-20250331-cfxs" src="https://github.com/user-attachments/assets/4bbd3090-6404-44f8-9771-1aea80fa56cc" />